tests/test_interface.rb

```diff
@@ -780,7 +785,8 @@ def test_negotiate_auto_ethernet
     rescue Cisco::CliError => e
       skip('Skip test: Interface type does not allow config change') if
         e.message[/requested config change not allowed/]
-      flunk(e.message)
+      flunk(e.message) unless
+        e.message[/cannot turn off auto-negotiate with auto-negotiate speeds/]
```

Comment:
Is it correct to ignore this message? Or should this trigger a skip like the above case?